### ISSUE TITLE
Evaluate the trained checkpoints and the provided checkpoints, and the results of the metrics are inconsistent

### ISSUE THREAD

**Issue by Guo-1212** [qianqianwang68/omnimotion#51]
Hello！
I made a modification to the default configuration as this comment（https://github.com/qianqianwang68/omnimotion/issues/37#issuecomment-1856324840）,There is a discrepancy between the metrics evaluated by the trained checkpoints and those provided.Take the blackswan dataset as an example，this is the metric measured with the checkpoints provided：
![屏幕截图 2024-02-22 170855](https://github.com/qianqianwang68/omnimotion/assets/149056694/c185c712-d05e-4582-a9f1-0422810b9df2)
This is the metric measured by the trained checkpoint：
![屏幕截图 2024-02-22 171544](https://github.com/qianqianwang68/omnimotion/assets/149056694/8d92476b-61f3-44a5-bc03-27a924fb429f)
In addition, the loss function of the training process also showed a downward trend and then an upward trend：
![屏幕截图 2024-02-22 171902](https://github.com/qianqianwang68/omnimotion/assets/149056694/ec8ca5a7-ccb9-46e1-9524-4b6cc7050035)
I would like to ask if there are any other configurations that need to be modified to present the results in the paper in addition to the changes involved in the above comment？
